### PR TITLE
Newsletters: add #18 (2018-10-23)

### PR DESCRIPTION
**Pull request by harding**
A bit wordy and technical this week; I'll try to be more concise and simple next week.

For the nmap results, I'm happy to email or IRC the commands to any of you who want to replicate the test (I just don't like posting exact port scanning commands publicly in order to reduce abuse).

### CODE REVIEW

File: _posts/en/newsletters/2018-10-23-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+
+        nmap -Pn -p 8332 ADDRESS
+
+    If the result in the *state* field is "open", you should follow the
+    instructions above to remove the `rpcallowip` parameter.  If the
+    result is either "closed" or "filtered", your node is safe unless
+    you've set a custom RPC port or otherwise have enabled a customized
+    configuration.
+
+    A [PR][Bitcoin Core #14531] has been opened to Bitcoin Core to make
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2018-10-23-newsletter.md]: D'oh, I thought I checked all the links.  Thanks!  Fixed (force-pushed).

File: _posts/en/newsletters/2018-10-23-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+    the extra pubkeys and signatures.
+
+    It will likely take time for experts to review these papers,
+    evaluate their security properties, and consider implementing
+    them---and some experts are already busy working on implementing a
+    consensus change proposal that would enable a Schnorr signature
+    scheme that would also provide for multiparty pubkeys and signatures
+    (and which also provides multiple other benefits).
+
+    - [Fast Multiparty Threshold ECDSA with Fast Trustless Setup][mpecdsa goldfeder] by Rosario Gennaro and Steven Goldfeder
```
[the remark this answers is not in the document]
Comment (harding) [reply _posts/en/newsletters/2018-10-23-newsletter.md]: Haha, you're right, it's an incongruity.  I was looking at the URL when choosing the link name and the paper is hosted on StevenGoldfeder.com.  :-)

File: _posts/en/newsletters/2018-10-23-newsletter.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (moneyball) [reply _posts/en/newsletters/2018-10-23-newsletter.md]: I still think it might be broken as the link is missing? This is what I see when I test it locally: "A [PR][Bitcoin Core #14532] has"

**Comment by jnewbery** [comment]
ACK 0efd8592a0faffe0162aa85324d475e8f03806ab
